util/tlgen/xbar.pkg.sv.tpl

```diff
@@ -16,7 +16,7 @@ package tl_${xbar.name}_pkg;
   % else:
   ## Xbar device
   localparam logic [${len(device.addr_range)-1}:0][31:0] ADDR_SPACE_${device.name.upper().ljust(name_len)} = {
-    % for addr in device.addr_range:
+    % for addr in list(reversed(device.addr_range)):
```

Comment:
@eunchan 
the reversing... i should probably leave a comment as to why this is done right?

Reply:
seems not change after using `reversed` in this file

Reply:
the main change happened [here](https://github.com/lowRISC/opentitan/pull/3276/files#diff-c6533ba21cbea223f5b8283e07088ce6R19)